client/state/sites/selectors.js

```diff
@@ -87,6 +88,30 @@ export function isJetpackSite( state, siteId ) {
 }
 
 /**
+ * Returns true if the Jetpack site is running a version meeting the specified
+ * minimum, or false if the Jetpack site is running an older version. Returns
+ * null if the version cannot be determined or if not a Jetpack site.
+ *
+ * @param  {Object}   state   Global state tree
+ * @param  {Number}   siteId  Site ID
+ * @param  {String}   version Minimum version
+ * @return {?Boolean}         Whether running minimum version
+ */
+export function isJetpackMinimumVersion( state, siteId, version ) {
+	const isJetpack = isJetpackSite( state, siteId );
+	if ( ! isJetpack ) {
+		return null;
+	}
+
+	const siteVersion = getSiteOption( state, siteId, 'jetpack_version' );
+	if ( ! siteVersion ) {
+		return null;
```

Comment:
In the use case here where the feature requires `4.1` or higher - won't returning `null` here cause a failure down the line in `EditorDrawerTaxonomies`?